config/manifests/bases/barbican-operator.clusterserviceversion.yaml

```diff
@@ -2,10 +2,15 @@ apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
   annotations:
-    alm-examples: '[]'
+    alm-examples: "[]"
     capabilities: Basic Install
     features.operators.openshift.io/disconnected: "true"
     features.operators.openshift.io/fips-compliant: "true"
+    features.operators.openshift.io/proxy-aware: "false"
+    features.operators.openshift.io/tls-profiles: "false"
+    features.operators.openshift.io/token-auth-aws: "false"
+    features.operators.openshift.io/token-auth-azure: "false"
+    features.operators.openshift.io/token-auth-gcp: "false"
     operators.openshift.io/infrastructure-features: '["disconnected"]'
```

Comment:
This is deprecated and can be removed

Reply:
Fixed in https://github.com/openstack-k8s-operators/barbican-operator/pull/131